scripts/ci.py

```
@@ -148,21 +148,26 @@ def cmd_quickstart(_args):
 
 
-def wait_for_health(base_url, timeout_secs=30):
+def wait_for_health(base_url, timeout_secs=120):
     url = f"{base_url.rstrip('/')}/healthz"
     step(f"Waiting for API to be ready at {url}")
     start = time.time()
+    attempt = 0
     while True:
         try:
+            attempt += 1
             with urlopen(url, timeout=1) as resp:
```

Ruff (0.14.13)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)